main.py

```diff
@@ -1,17 +1,20 @@
-import sys
-import argparse
+import argparse, sys
+from calculator.operationCreators import CreateSummation, CreateSubtraction
 
 
-def main() -> int:
-    """Main function..."""
-    parser = argparse.ArgumentParser()
-    parser.add_argument("-v", "--value", dest="val", type=int, help="Value to use.")
-    args = parser.parse_args()
-
-    print(args.val)
+def main(args) -> int:
```

Comment:
```suggestion
def main(args) -> Union[int, float]: 
```
Might fit better for future purposes.